devtools/common/src/main/filtered/extensions.json

```diff
@@ -403,5 +403,15 @@
     ],
     "groupId": "io.quarkus",
     "artifactId": "quarkus-flyway"
+  },
+  {
+    "name": "Keycloak",
```

Comment:
I think we can add `Keycloak` extensions by following alphabetical order as suggested by this [PR](https://github.com/quarkusio/quarkus/pull/1856)